src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -285,7 +285,49 @@ static void add_pending_ping(pending_ping **root, grpc_closure *notify) {
  * glb_lb_policy
  */
 typedef struct rr_connectivity_data rr_connectivity_data;
-static const grpc_lb_policy_vtable glb_lb_policy_vtable;
+
+/* Forward declare functions referred in glb_lb_policy_vtable */
```

Comment:
I generally prefer to avoid forward declarations whenever possible, since they require maintaining two copies of the function signatures.  Please move the vtable back where it was and remove the forward declarations.

Reply:
This would require rearranging the functions, so that it finds the declarations.

Reply:
Why?  This was working fine before this PR.  Is there some reason that the original code would not build as C++?

Reply:
Oh. C++ does not allow static const declarations without any value being assigned

Reply:
Ah, okay.

It looks like the only thing that references `glb_lb_policy_vtable` between here and where it was originally defined below is `glb_create()`.  So I suggest simply moving `glb_create()` from its original position down to right after the definition of `glb_lb_policy_vtable`.

Reply:
done